ipn/ipn_clone.go

```diff
@@ -1,9 +1,9 @@
-// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
 // Code generated by tailscale.com/cmd/cloner; DO NOT EDIT.
-//go:generate go run tailscale.com/cmd/cloner -type=Prefs -output=prefs_clone.go
+//go:generate go run tailscale.com/cmd/cloner -type=Prefs
```

Comment:
did I already lose this battle once? I don't like go:generate in generated files. I like being able to nuke them and "go generate" them back into existence, and have the go:generate line at the source of truth (the input data to the generation)